### DISCUSSION TITLE
custom Gcode for PAUSE Print. [FR]

### DISCUSSION THREAD

**Discussion by teveo-1** [supermerill/SuperSlicer#1337] [Community contribution]
1st : thanks for that great slicer !
2 : is it possible to implement in the "printer settings" the possibility to add a  special Gcode for the pause ( like the Start G-code, End g-code, etc...)  ...With the 2.3.56,  I can add my own gcode for the pause but I have to copy/paste  directly in the slicer (...right click on the small "+"  ,  add custom G-code (small G). 
Thanks.

**Comment by supermerill** [comment]
create an feature request issue for feature request.

**Reply by neophyl** [reply]
I thought that would already be in Super Slicer as its part of PS 2.3.  The custom gcode gui has sections for Colour Change, Pause Print and Template Custom that allows the user to define whats inserted when you use the right click on the layer slider.

Does that section exist in Super Slicer but just not part of the gui ?
